src/nl/surf/eduhub/validator/service/main.clj

```diff
@@ -44,17 +45,31 @@
 (def opt-specs
   {:gateway-url             ["URL of gateway" :str :in [:gateway-url]]
    :gateway-basic-auth-user ["Basic auth username of gateway" :str :in [:gateway-basic-auth :user]]
-   :gateway-basic-auth-pass ["Basic auth password of gateway" :str :in [:gateway-basic-auth :pass]]})
+   :gateway-basic-auth-pass ["Basic auth password of gateway" :str :in [:gateway-basic-auth :pass]]
+   :introspection-client-id ["Basic auth username of introspection" :str :in [:introspection-basic-auth :user]]
+   :introspection-secret    ["Basic auth password of introspection" :str :in [:introspection-basic-auth :pass]]
+   :introspection-endpoint  ["Introspection endpoint url" :str :in [:introspection-endpoint-url]]})
```

Comment:
1. De naamgeving van deze configuratie variabelen is denk ik niet duidelijk genoeg. 

In de rio mapper gebruiken we:

```
SURF_CONEXT_CLIENT_ID               SurfCONEXT client id for Mapper service
SURF_CONEXT_CLIENT_SECRET           SurfCONEXT client secret for Mapper service
SURF_CONEXT_INTROSPECTION_ENDPOINT  SurfCONEXT introspection endpoint
```
Dat is denk ik duidelijker voor degene die dit moet configureren

2. Deze info moet ook de in README staan